src/workflows/utils/labels.config

```diff
@@ -43,24 +46,3 @@ process {
   // withLabel: highdisk { disk = { 200.GB * task.attempt } }
   // withLabel: veryhighdisk { disk = { 500.GB * task.attempt } }
 }
-
```

Comment:
Just a note: defining functions inside the config is deprecated; hence the repetition of code...